### PR TITLE
Time Converter Automations in Branch1

### PR DESCRIPTION
**Pull request by nghi01**
## Assigned Issue Number ? { if does not exist then create one }
#58 
## Language Opted ?
C++

## Number of files added or updated ?
3

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've referred the correct issue number.
- [x] I've fill up this entire template correctly.

### CODE REVIEW

File: Automations/TimeConverter/timeconverter.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    std::cout << "This is the Time converter.\n";
+    while (check1)
+    {
+        //Ask for 1st unit of time
+        std::cout << "Please input the original unit of time that you want to convert.\n";
+        std::cout << "Valid inputs are: SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, YEAR, DECADE, CENTURY.\n";
+        std::cin >> originalUnit;
+        std::transform(originalUnit.begin(), originalUnit.end(),originalUnit.begin(), 
+                    [](unsigned char c) -> unsigned char { return std::toupper(c);});
+        string1 = convert(originalUnit);
```
Comment (Arsenic-ATG) [inline Automations/TimeConverter/timeconverter.cpp:50]: shouldn't the names of `string1` and `originalUnit` be interchanged ?

Comment (Arsenic-ATG) [inline Automations/TimeConverter/timeconverter.cpp:50]: shouldn't the names of `string1` and `originalUnit` be interchanged ?

File: Automations/TimeConverter/timeconverter.cpp [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }
+    while (check2)
+    {
+        //Ask for 2nd unit of time
+        std::cout << "Please input the converted unit of time that you want to convert.\n";
+        std::cout << "Valid inputs are: SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, YEAR, DECADE, CENTURY.\n";
+        std::cin >> convertedUnit;
+        std::transform(convertedUnit.begin(), convertedUnit.end(), convertedUnit.begin(), 
+                    [](unsigned char c) -> unsigned char { return std::toupper(c);});
+        string2 = convert(convertedUnit);
```
Comment (Arsenic-ATG) [inline Automations/TimeConverter/timeconverter.cpp:69]: same for `string2` and convertedUnit`

Comment (Arsenic-ATG) [inline Automations/TimeConverter/timeconverter.cpp:69]: same for `string2` and convertedUnit`

File: Automations/TimeConverter/enum.cpp [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include <string>
+
+enum Time {SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, YEAR, DECADE, CENTURY, NONE};
+
+Time convert(std::string thisString)
```
[the remark this answers is not in the document]
Comment (nghi01) [reply Automations/TimeConverter/enum.cpp:5]: Well, I think it's possible to include in the main file. However, I prefer leaving it here as a data-like file to configure.

File: Automations/TimeConverter/timeconverter.cpp (hunk 1 above)
Comment (nghi01) [reply Automations/TimeConverter/timeconverter.cpp:50]: yes it's my bad I'll fix it.

Comment (nghi01) [reply Automations/TimeConverter/timeconverter.cpp:50]: Yes, I will fix it.

File: Automations/TimeConverter/timeconverter.cpp (hunk 2 above)
Comment (nghi01) [reply Automations/TimeConverter/timeconverter.cpp:69]: Same

Comment (nghi01) [reply Automations/TimeConverter/timeconverter.cpp:69]: Same

**Comment by nghi01** [comment]
What is this check that I'm failing, could you explain more about the reason why it failed?

**Comment by Arsenic-ATG** [comment]
> What is this check that I'm failing, could you explain more about the reason why it failed?

Oh, it have nothing to do with this PR, you can safely ignore it.

----
That's due to an erroneous commit done earlier by me, which accidentally changed the EINTEGRITY value of one of the js packages, causing the checksum to fail. I would be fixing it soon.

**Comment by nghi01** [comment]
I've fixed the code in my branch, do I create a new pull request or what should I do?

**Comment by Arsenic-ATG** [comment]
> I've fixed the code in my branch, do I create a new pull request or what should I do?

GitHub is smart enough to update the existing PR if there are any changes in the base branch so there is no need of creating new PR.

Thanks for the contribution 🙂👍

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**